airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -178,7 +178,7 @@ definitions:
         $ref: "#/definitions/requester"
         request_parameters:
           "fields[metric]": "name,created,updated,integration"
-          "include": "metric"
+          "include": "metric,attributions"
```

Comment:
mock server tests like `test_read_records_events` seem to be failing because of this addition. Can we fix them?